test/oComponentBoilerplate.test.js

```diff
@@ -0,0 +1,7 @@
+const oComponentBoilerplate = require('./../main');
```

Comment:
it should require the actual module. Main just requires the module and listens to o.DOMContentLoaded.